### ISSUE TITLE
Arranging accessibility metadata in two levels.

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/publ-a11y#136]
One of the important recommendations received from the user groups is to arrange the accessibility metadata information in the user experience guide in two levels.
* Level 1: High level overview with some icons to help people with cognitive limitations.
* Level 2: more detailed information. It should explain what user can do.
* The third level should be raw metadata (either a link should be provided to raw metadata or it can be put in HTML details)
We should be careful for selecting information to be presented in level 1, because it is possible that some retailers display only level 1 information.

Currently the user experience guide recommends the information in the following order. Please suggest how it can be placed in level 1 and level 2.
* Screen Reader Friendly
* Full Audio (if present)
* Accessibility Summary
* Accessibility Conformance
* Certified By
* Certifier’s Credential
* Certifier’s Report (if present)
* Hazards
Some more metadata infomation may be added from issue #137 
User experience guide is at:

https://www.w3.org/2021/09/UX-Guide-metadata-1.0/principles/

**Comment by dazrand** [comment]
Level 1. Ideally this would carry all the ones listed above. Assuming most retailers will limit what is listed, I suggest the most important would be Screen Reader Friendly, Hazards, Format (from #137), and Accessibility Summary. Ideally I would include all accessModeSufficient listings to give a better breakdown, but this is basically covered by Screen Reader. 

Level 2. Assuming a level 3 is present with the raw metadata listing everything, I could see Level 2 being limited to Conformance and Credential entries.

**Comment by gautierchomel** [comment]
First outcomes from french initiative (54 organizations): 

Before accessibility information, book details should at least show:
* file format
* DRM or absence of
* main language 

At level 1 reading modes, critical features and hazards should be present: 
* _Text and images available in synthetic voice and Braille_ or _audiobook_
* _Adaptable display_ or _fixed layout_
* _synchronized text & audio_
* _Reference to the pagination of printed books_
* _MathML _ or _ described math_ 
*  Hazards (if presents)

At level 2 : 
* _Graphic resources are described in detail_
* Conforms To
* Conformance report 
  * (if conformance report is present and displayed as a link, other conformance information will be found there)
  * (if conformance report is not present, then display here _certified by_ and _certifier credential_)
  * Accessibility Summary

At level 3 : 
* any other accessibility metadata

**Comment by HadrienGardeur** [comment]
We're implementing support for accessibility metadata across our products right now and this is our approach:

- Fully or partially accessible (we evaluate against various values for WCAG level A or AA+)
  - Conformance details (specific profile)
  - Accessibility summary
  - Credentials
  - Report
- Screen-reader and TTS friendly (we evaluate the sole presence of `textual` in `accessModeSufficient`)
- Full audio (we evaluate the sole presence of `auditory` in `accessModeSufficient`)
- Synchronized text and audio (we evaluate the presence of `synchronizedAudioText` in `accessibilityFeature`)
- Customizable text and layout (we evaluate the presence of `displayTransformability` in `accessibilityFeature`)
- Table of contents navigation (we evaluate the presence of `tableOfContents` in `accessibilityFeature`)
- Heading navigation (we evaluate the presence of `structuralNavigation` in `accessibilityFeature`)
- Described images (we evaluate the presence of `alternativeText` or `longDescription` in `accessibilityFeature`)
- Described math (we evaluate the presence of `describedMath` in `accessibilityFeature`)
- Hazards (if any)
- Other accessibility features (if any)

**Comment by gautierchomel** [comment]
Workshops in France also raised that Conforms To display as it is proposed is too technical and that a human understandable statement would add fluidity for readers. 
We propose a combination of label and technical reference as per #138

**Comment by gautierchomel** [comment]
Yes, _Conformance_ says a lot and displaying each feature of the conformance is heavy and redundant. 

But if I'm partially compliant I can't add _Conformance_ and I want to list features presents. Anyway if I display them all, as a user, I won't be able to identify the missing feature for conformance. 

In that case actually I see _AccessibilitySummary_ as my best friend.

**Comment by wareid** [comment]
Yeah I agree, if you're not conformant but do have features it would be good to provide them.

**Comment by avneeshsingh** [comment]
I think that it is OK to have some duplicate information. Our target audience is end users, and not the technical community. So, we can relax the logic of avoiding redundant information that we use in technical world. Some users will understand from conformance information, while some users would like to see explicit mention that publication works well with screen reader or it has some hazards which are not good for them.

On the other side, we should also assume that retailers who would be displaying this information have limited information about accessibility.

Third important point to highlight is that AccessibilitySummary is not a summary of accessibility metadata. Actually the name is little misleading. AccessibilitySummary provides the additional information which is not conveyed by machine readable accessibility metadata. Which means if AccessibilitySummary is provided, it has some important information which user should know.

**Comment by HadrienGardeur** [comment]
We consider that this is useful inference rather than duplication.

For end-users, knowing that a book is "fully accessible" or "conforms to WCAG 2.0 AA" might be completely meaningless, whereas pointing out that the book will work well with screen readers or TTS could be better understood.

It's more difficult to infer that a book has alternative text or full audio since it requires to check against both `accessMode` and `conformsTo` or to inspect the content of the file. 

In our case, we have a mix of inferences based on accessibility metadata and content.

**Comment by clapierre** [comment]
I strongly feel that conformsTo, certifiedBy and certifiersCredentials need to be at Level 1.  Knowing an EPUB is conformant to the EPUB Accessibility 1.1 and WCAG 2.x A(A) is extremely important and knowing who certified it and if they are credentialed will help those in charge of purchasing accessible materials, so they don't have to dig to find out if this EPUB has been certified by a reputable 3rd party.

**Comment by gregoriopellegrino** [comment]
As we worked on this issue, we realized that perhaps it might be useful-before defining the order in which to display metadata-to reason about some different possibilities for grouping metadata. Find our proposal at issue #139 .

**Comment by avneeshsingh** [comment]
November 24, 2022 meeting was dedicated to this topic. Please go through [the minutes](https://github.com/w3c/publ-a11y/wiki/Meeting-minutes-of-Publishing-CG-Accessibility-Task-Force).

**Comment by gregoriopellegrino** [comment]
Working on issue #139 we identified these levels:

> * first level (perhaps placed together with other important information, such as price): **Accessibility Compliance** with local laws, e.g., for Europe "European Accessibility Act compliant" (as discussed in issue [Refine Conforms To display #138](https://github.com/w3c/publ-a11y/issues/138))
> * second level: **Accessibility Properties**, organized into _Accessible navigation_, _Accessible non-textual content_, _Visual Adjustments_, _Audio_, _Hazards_ (in summary form and with extended description)
> * third level: **Accessibility Certification**
> * fourth level: **Additional information on accessibility**, e.g. accessibility summary, other metadata
> 
> Generic metadata (issue #137) in our opinion should be included on the product page, along with other metadata about the ebook, but not within the accessibility section.

**Comment by gautierchomel** [comment]
We've gone thru a protoype / iteration process with stakeholders and end users. We are now going thru a production testing with two online bookshops, I hope to have some survey results by the end of january.

The main difficulty we meet for this step is the lack of information's present in ONIX files. It also raises difficulties about quality of the present information. This has to be taken in count: few good information is more understandable for users and for content providers.

Also to mention there is a similar research project with end users in Netherlands. Results are expected for January.

**Comment by LaZay** [comment]
Here is my proposal for a11y properties distribution upon levels 1 / 2 / 3.

LEVEL 1 = a11y general information
accessibilitySummary
accessModeSufficient
dcterms:conformsTo
Digital protection
- accessibilityFeature - unlocked
- watermarking
- DRM LCP

LEVEL 2 = a11y functional details
Accessibility feature - readingOrder
Accessibility feature - structuralNavigation
Accessibility feature - tableOfContents
Accessibility feature - printPageNumbers
Accessibility feature - MathML
Accessibility feature - alternativeText
Accessibility feature - caption
Accessibility feature - transcript
Accessibility feature - audioDescription
Accessibility feature - synchronizedAudioText
AccessibilityControl - fullKeyboardControl
Hazards - noFlashingHazard
Hazards - noMotionSimulationHazard
Hazards - noSoundHazard

LEVEL 3 = a11y reliability of conformance to standards 
a11y:certifierReport
a11y:certifiedBy
a11y:certifierCredential

**Comment by gautierchomel** [comment]
Instead of levels we used categorization and proposed compact and descriptive sentences. 

It stays that 3 key information are considered as necessary to be always present (see issue #184).
